app/main.py

```diff
@@ -1 +1,55 @@
-# write your code here
+class SoftwareEngineer:
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills += [skill]
+
+
+class FrontendDeveloper(SoftwareEngineer):
+
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.skills = ["JavaScript", "HTML", "CSS"]
```

Comment:
```suggestion
        self.skills += ["JavaScript", "HTML", "CSS"]
```

Reply:
you must add skills in init to unite all skills if class inherits from 2 or more class

Reply:
do it in every class